# message that gets published by the gui

std_msgs/Header header


bool start_stiffness
bool start_experiment
bool start_logger
bool learning
uint16 experiment_number


